src/cljs/main/broadfcui/common/notifications.cljs

```diff
@@ -71,7 +71,6 @@
    :-load-service-alerts
    (fn [{:keys [this]} & [first-time?]]
      (utils/ajax {:url (config/alerts-json-url)
-                  :headers {"Cache-Control" "no-store, no-cache"}
```

Comment:
Removed because this has no effect and was misleading during debugging
> Note: Cache-Control is also a header you can specify in HTTP requests for an object. Cloud Storage ignores such headers if they are set to avoid cached content.

[Source.](https://cloud.google.com/storage/docs/metadata?hl=en_US&_ga=2.170911328.-1166978692.1449000140#cache-control)